Updated Values

Remy-Reveltek opened the pull request:
- Updated input fields for last name, email, and phone no.
- Changed website to phone in script.js

Remy-Reveltek committed:
Updated Values

- Updated input fields for last name, email, and phone no.
- Changed website to phone in script.js

bjornmage reviewed (changes requested):
Change script.js to have first name and last name variables to match our form.

Remy-Reveltek commented:
Ok

Remy-Reveltek committed:
Updated Script.js

- Changed script.js to have first name and last name variables to match our form.
- Added URL of form `https://door.taolearning.org/api/form`

Remy-Reveltek commented:
Updated required changes:
- Changed script.js to have first name and last name variables to match our form.
- Added URL of form `https://door.taolearning.org/api/form`

Remy-Reveltek committed:
Cleaned up the code
